app/Http/Controllers/MovieController.php

```diff
@@ -29,9 +29,9 @@ public function __construct()
             'update'
         ]);
 
-        $this->middleware('can:view,movie')->only([
-            'getActors'
-        ]);
+        // $this->middleware('can:view,movie')->only([
+        //     'getActors'
+        // ]);
     }
```

Comment:
Is this zombie code? Can it be removed permanently?

This data is no longer protected - I think we should put the middleware back in.